frontend/src/metabase/qb/components/actions/CountByTimeAction.jsx

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+
```

Comment:
These drill-through actions could have short explanations in the beginning of file well